sig-cloud-provider/2020-annual-report.md

```diff
@@ -0,0 +1,85 @@
+
+I believe so.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+1. We look at the number of people attending our meetings, messaing on our slack channels and contributing to our repos.
+1. We look at the number of cloud providers who have people doing the above.
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+1. For changes to cloud provider infrastructure where all approvers can approve, I think things are going reasonably well. But I'd like to hear what others think.
+1. For changes to a specific cloud provider implementation, where an approval is required from someone outside the company, I think there is a bigger problem, particularly for VMWare and Google since the SIG Chairs (the most active approvers) are from those respective companies. I feel like we need to work toward having more approvers (e.g. get to the point where the defacto bus factor is at least 2 for any change). I don't think this necessarily requires more top level approvers in the short term-- getting more sub-tree approvers (with good diversity across cloud providers) and expanding their ownership seem like a good steps.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+Most of our SIG members are contributors from at least one cloud provider. We try to reach our to the various cloud providers during KubeCon to make sure they are aware of us and that their voices are being heard. New members come in slowly enough that we can generally provide direct mentorship and arrange this through our slack channel or meetings.
+
+**What programs do you participate in for new contributors?**
+
+None, our contributors tend to be representatives from the various companies who provider Kubernetes on the Cloud. It would be nice to expand this however.
+
+**Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?**
+
+We try hard to make sure we have presentation from each of the interested cloud providers.
+
+---
+## Current initiatives and project health
+_Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members._
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+[Feature: implement the BackendManager list](https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/144) fixes a long standing issue for people running clusters with nodes which are either not routable or at least not close to one another.
```

Comment:
are all of these long tail or things you'd like to highlight as awesome work?

Reply:
highlighted as awesome work